src/main/java/model/figure/Square.java

```diff
@@ -0,0 +1,23 @@
+package model.figure;
+
+import model.Figure;
+
+public class Square extends Figure {
```

Comment:
The class implementation should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.